database/migrations/001-punishments-schema.sql

```diff
@@ -0,0 +1,46 @@
+--------------------------------------------------------------------------------
+-- Up
+--------------------------------------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS punishments (
+	punishType TEXT,
+	userid TEXT,
+	ips TEXT,
+	userids TEXT,
+	expireTime INTEGER,
+	reason TEXT,
+	-- The column rest is used to make the punishments database extensible
```

Comment:
I understand this is legacy, but in a database world would we not just create a migration to change the schema if we need to extend it?